packages/canary-web-components/src/components/ic-data-table/ic-data-table.tsx

```diff
@@ -40,7 +40,9 @@ import {
   dynamicDebounce,
   getSlotElements,
   checkResizeObserver,
+  deviceSizeMatches,
 } from "../../utils/helpers";
+import { IC_DEVICE_SIZES } from "@ukic/web-components";
```

Comment:
this import is failing. I think it needs to be: 
```suggestion
import { IC_DEVICE_SIZES } from "../../utils/constants";
```